torch/fx/subgraph_rewriter.py

```diff
@@ -193,14 +204,16 @@ def replace_pattern_with_filters(
     pattern: Union[Callable, GraphModule],
     replacement: Union[Callable, GraphModule],
     match_filters: List[Callable[["InternalMatch", Graph, Graph], bool]],  # type: ignore[name-defined]
-) -> List[Match]:
+) -> List[ReplacedPatterns]:
     """
     See replace_pattern for documentation. This function is an overload with an additional match_filter argument.
 
     Args:
         ``match_filter``: A function that takes in (match: InternalMatch, original_graph: Graph, pattern_graph: Graph)
             and returns a boolean indicating whether the match satisfies the condition. See matcher_utils.py for
             definition of InternalMatch.
+        ``get_replacement_nodes``: If set, causes the function to return a list
```

Comment:
do we still need this flag? 
We can return replacementPattern by defatult.

Reply:
oh yes, sorry forgot to remove that comment.

Reply:
ooh yes, sorry forgot to remove that comment